pkg/app/app.go

```diff
@@ -1369,11 +1369,16 @@ func (a *App) apply(r *Run, c ApplyConfigProvider) (bool, bool, []error) {
 		PostRenderer:      c.PostRenderer(),
 	}
 
+	args := strings.Join([]string{c.Args(), c.DiffArgs()}, " ")
```

Comment:
add some comments for this logic. thanks so much.

Reply:
```suggestion
    # join --args and --diff-args together to one string.
	args := strings.Join([]string{c.Args(), c.DiffArgs()}, " ")
```